components/CreateForm.tsx

```diff
@@ -0,0 +1,128 @@
+import React, {useState} from "react";
+import AutoComplete from "react-google-autocomplete"
+import { FormEvent } from "react";
+import { useSession } from "next-auth/react";
+import { Session } from "next-auth";
+
+interface postData {
+    title: string,
+    latitude: number,
+    longitude: number,
+    ownerId: string,
+    description: string,
+    availableSpots: number,
+    email: string | null | undefined,
+    name: string | null | undefined
+    eventTime: string
+}
+
+export interface props {
+    session: Session
+    name: {
+        name: string
+        id: string
+    }
+}
+
+const CreateForm: React.FC<props> = (props) => {
```

Comment:
Rename this

Reply:
Do I change this to 'CreateFormPost' ?

Reply:
This is a form, so I it would make more sense for it to be called `CreatePostForm`

Reply:
Changed name to `CreatPostForm`